src/main/java/com/entropyteam/entropay/employees/services/EmployeeService.java

```diff
@@ -216,7 +225,18 @@ private boolean shouldDeactivateEmployee(UUID employeeId, Employee entityToUpdat
         return existingEmployee.isActive() && !entityToUpdate.isActive();
     }
 
-    public String getEmployeesTimeSinceStart(Contract firstContract, Contract latestContract){
+    public CalendarEventDto formatEventData(UUID employeeId, LocalDate birthDate, String firstName, String lastName) {
+        int currentYear = LocalDate.now().getYear();
+        LocalDate startDate = birthDate.withYear(currentYear);
+        LocalDate endDate = startDate.plusDays(1);
+        String eventId = currentYear + "-" + employeeId.toString();
```

Comment:
You are adding a "-" here, then replacing it for a blank space in the google service. Lets change to:
`String eventId = currentYear + employeeId.toString();`